scapy/volatile.py

```diff
@@ -252,7 +252,12 @@ def __mul__(self, n):
 
 class RandBin(RandString):
     def __init__(self, size=None):
-        RandString.__init__(self, size, "".join(map(chr, range(256))))
+        RandString.__init__(self, size, b"".join(map(chb, range(256))))
```

Comment:
I think it would be better to use `super()` here. Also, we could remove `map()`, and we should name the named arguments:
```python
    def __init__(self, size=None):
        super(RandBin, self).__init__(size=size, chars=b"".join(chb(c) for c in range(256)))
```